Makefile

```diff
@@ -26,7 +26,13 @@ DOC_FILES := \
 	project.md \
 	media-types.md \
 	manifest.md \
-	serialization.md
+	descriptor.md \
+	image-layout.md \
+	serialization.md \
+	layer.md \
+	GOVERNANCE.md \
+	RELEASES.md
```

Comment:
I debated about those two because they are some what boiler plate. I'll
remove them.

On Wed, Sep 7, 2016 at 2:47 PM W. Trevor King <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/image-spec/pull/275#discussion_r77881516>
> :
>
> > @@ -26,7 +26,13 @@ DOC_FILES := \
> >  	project.md \
> >  	media-types.md \
> >  	manifest.md \
> > -	serialization.md
> > +	descriptor.md \
> > +	image-layout.md \
> > +	serialization.md \
> > +	layer.md \
> > +	GOVERNANCE.md \
> > +	RELEASES.md
>
> Do we need these last two? I think they're metadata, and not part of the
> spec itself. runtime-spec does not include them in the generated spec
> <https://github.com/opencontainers/runtime-spec/blob/22c29c18eddc75ccee9f6b202f61524f29653c5f/Makefile#L23-L38>
> or link to them from any files which are part of the generated spec.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/275/files/13755da234e41fb12131570e58514f5ef7ab7866#r77881516>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEF6Y_jyR2IsQVGlvy89qnc9-Mey352ks5qnwbPgaJpZM4J3Oq0>
> .
>